lib/gamepiece.rb

```diff
@@ -0,0 +1,8 @@
+class GamePiece
```

Comment:
What's the use of this GamePiece class? Please explain in a comment below so. I may direct you better.
It seems like there should be methods here associated with a Game piece? 
What was your intention for this class?